joomla-projects/GSoC17_publishing_workflow
# Implementing permissions for workflows

Pull Request for Issue #138 .

### Summary of Changes

This implementation is to validate user's permission in workflow component. Similar to categories, workflow permission uses it's parent component's permission settings plus its own.
A new permission transition.run is used to indicate which users can perform a particular transition.

### Testing Instructions

Setup Joomla CMS. Create user groups and change permissions. Create users for those groups, and check the visibility of different operations based on permissions
